tekton/koparse/test_koparse.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3.6
+
+import os
+import unittest
+
+import koparse
+
+
+IMAGE_BASE = "gcr.io/knative-releases/github.com/knative/build-pipeline/cmd/"
+PATH_TO_TEST_RELEASE_YAML = os.path.join(os.path.dirname(
+    os.path.abspath(__file__)), "test_release.yaml")
+PATH_TO_WRONG_FILE = os.path.join(os.path.dirname(
+    os.path.abspath(__file__)), "koparse.py")
+BUILT_IMAGES = [
```

Comment:
We need `cmd/entrypoint` too :angel:

Reply:
actually the test data (which is the 0.1 release) doesnt have the entrypoint image! 😇 

but this reminds me, i forgot to address @pivotal-nader-ziada 's comment and add the entrypoint image to the ko.yaml so I'll do that now 🤦‍♀️ 😅